owners_generate.go

```diff
@@ -26,7 +26,7 @@ func main() {
 	}
 
 	// each of the plugins, in case someone is not in the top-level one
-	err = filepath.Walk("plugin",
+	_ = filepath.Walk("plugin",
```

Comment:
keep the `err` and `log.Fatal(err)` afterwards

Reply:
updated